setup.py

```diff
@@ -40,11 +40,10 @@
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
     ],
-    install_requires=[
-        'simplejson >= 2.0.9'],
     setup_requires=[
         'versiontools >= 1.3.1'],
     tests_require=[
+        'simplejson >= 2.0.9',
```

Comment:
The tests 'work', but mostly fail because the messages are different from the expected messages.